python/ray/serve/tests/test_standalone.py

```diff
@@ -145,7 +145,6 @@ def connect_in_deployment(*args):
     assert "deployment-ception" in serve.list_deployments()
 
 
-@pytest.mark.skipif(sys.platform == "win32", reason="Failing on Windows")
```

Comment:
What's the hardware configuration (number of CPUs, RAM) used by buildkite? @simon-mo

Reply:
4cpus 8g mem

Reply:
This test passes in 75 - 80s with 4 CPUs and 5 GB RAM. So, I think we are good with unskipping this test for now. cc: @pcmoritz